File: Python/Наибольший элемент массива и его номер.py
from random import random, randint
max=0
max1=0
a=['']*20
c=0
k=0
for i in range (20):
    a[i]=randint(0,20)
    max=a[i]
    if max1<max:
        max1=max
        g=i
print(a)
print(max1)
print(g)

File: C++/Наибольший элемент массива и его номер.cpp
#include<iostream>
#include<cstdlib>
#include<ctime>
using namespace std;
int main(){
	srand(time(NULL));
	setlocale(LC_ALL, "Russian");
	int m[20];
	int max=0;
	int max1=0;
	int g=0;
	for (int i=0; i<20; i++){
		m[i]=rand()%30-10;
		cout<<m[i]<<' ';
		max=m[i];
		if (max1<max){
			max1=max;
			g=i;
		}
	}
	cout<<endl<<max1<<endl<<g;
}
